app/services/expired_registrations_service.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class ExpiredRegistrationsService < ::WasteExemptionsEngine::BaseService
+  def run
+    all_expired_registration_exemptions.update_all(state: :expired)
```

Comment:
Can we use `.expire!` to use the state machine instead of directly updating the value in the DB?

See https://github.com/DEFRA/waste-exemptions-back-office-ta/blob/master/app/models/concerns/can_deactivate_exemption.rb#L27-L29